### DISCUSSION TITLE
Generative AI Use Cases in IoT Applications

### DISCUSSION THREAD

**Discussion by nj2208** [thingsboard/thingsboard#11281] [Q&A]
Hello, everyone!

I 'm curious to know: can Generative AI enhance IoT applications beyond traditional analytics and monitoring?

Have any of you implemented or considered implementing GenAI in your IoT projects?

What specific use cases have you found GenAI to be particularly effective for in IoT environments?

Please share your thoughts. Thank you in advance for your contributions!

Regards

PS: Found this article for some uses cases of GenAI in IoT Context.

https://www.forbes.com/sites/forbestechcouncil/2024/04/10/leveraging-generative-ai-and-llms-in-the-industrial-iot-realm/

**Comment by Mdmartinez97** [comment]
[OpenAI Vision 1](https://www.youtube.com/watch?v=gZp9B_IiKCo)
[OpenAI Vision 2](https://www.youtube.com/watch?v=N1f-wzB7Qwc)

Just a vague idea... I think that by using computer vision (you don't even need to show what the camera sees), you can do away with many sensors and just ask the AI ​​what a particular system should do.
This could be an opportunity...
